### PR TITLE
Multisite reviews

### PR DESCRIPTION
**Pull request by TsankoBg**
This PR adds support for syncing product reviews to [Wordpress Multisite](https://developer.wordpress.org/advanced-administration/multisite/create-network/). In wordpress multisites, the product IDs are not unique.

[sc-37278]

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+= 3.33 - 2024-01-23 =
+* Support review sync in Multisite setup
```
Comment (apeschar) [inline changelog.md]:
```suggestion
* Support review sync in multisite setup.
```

**Review by apeschar — COMMENTED** [review]
This is a bit of a hack.

The proper solution would be to make sure that for multisite setups the correct site is used for the sync (the same site that the product came from). But I don't recall whether we even have support for synchronizing reviews to multiple sites.

I'll merge this now, because I guess it's not worth investing much time in.

**Comment by apeschar** [comment]
✅ Released as 3.33.

**Comment by TsankoBg** [comment]
> This is a bit of a hack.
> 
> The proper solution would be to make sure that for multisite setups the correct site is used for the sync (the same site that the product came from). But I don't recall whether we even have support for synchronizing reviews to multiple sites.
> 
> I'll merge this now, because I guess it's not worth investing much time in.

👍  agree. 

I did not find a better way to match the product to the origin site. I was thinking of comparing the product URL and the site URL, but I think this is even worse. The other idea is to send `sync back URL`  API endpoint and save it to every product

**Comment by apeschar** [comment]
> The other idea is to send sync back URL API endpoint and save it to every product

Yeah, that would have to be it, I guess. Because of course every product ID is only valid on a particular site. Multi-site WordPress installs are basically separate sites on the same domain, and should be treated as such.

**Pull request merged**